api/import-certs.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+CERT_DIR="/etc/kafkaui/certs"
+KEYSTORE="$JAVA_HOME/lib/security/cacerts"
```

Comment:
I'd rather see a custom truststore created and passed via these spring properties
```
server:
  ssl:
    trust-store: classpath:keycloak-truststore.jks
    trust-store-password: changeit
```
rather than messing with jre's truststore, what do you think?

Reply:
where configure spring properties?
do I need to add a certificate to keycloak-truststore.jks ?

Reply:
>do I need to add a certificate to keycloak-truststore.jks ?
we can build a new truststore from scratch within the same script and put it somewhere

>where configure spring properties?
I believe we can try setting env vars like `SERVER_SSL_TRUST-STORE: xxx`, or leave this to the user (given this will be well documented, adding a few config properties is way easier than building a truststore from scratch)

Reply:
I am not Java developer. I don`t know spring. Could you share simple example without kubernetes?

Reply:
@Haarolean Could you share simple example without kubernetes?

Reply:
Sorry for the delay. This is a config property, it has nothing to do with k8s. 
My suggestion is to create a truststore from scratch and have it mounted into the container, so the user can set the location via `SERVER_SSL_TRUST-STORE` themselves. What do you think?

Reply:
I would like to just specify the k8s secret or k8s configmap with the certificate, and it created the certificate store itself inside